source/_patterns/02-organisms/global/site-header.mustache

```diff
@@ -1,21 +1,28 @@
 <header class="site-header clearfix" data-behaviour="SiteHeader" id="siteHeader">
-  {{#title}}
-    {{> molecules-site-header-title }}
-  {{/title}}
-  <div class="site-header__navigation" role="navigation" aria-label="Main navigation">
 
-    {{#secondaryLinks}}
-      {{> molecules-site-header-nav-bar }}
-    {{/secondaryLinks}}
+    <div class="site-header__navigation" role="navigation" aria-label="Main navigation">
+        <div class="site-header__navigation__item">
+            {{#title}}
+                {{> molecules-site-header-title }}
+            {{/title}}
 
-    {{#primaryLinks}}
-      {{> molecules-site-header-nav-bar }}
-    {{/primaryLinks}}
+            <span class="site-header__border"></span>
```

Comment:
I think this is not relevant, because I'm reusing "molecules-site-header-title" in a lot of components and I need that class "site-header__border" only in that file. If I move that class name to another element inside that header title, it will affect all of the component, using "molecules-site-header-title".

Reply:
I will have a look at this later as I do not really want a span that is empty.